fjord/suggest/utils.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from fjord.suggest import get_providers
+
+
+logger = logging.getLogger('i.suggest')
+
+
+def get_suggestions(response):
+    """Given a feedback response, returns an ordered list of suggestions"""
```

Comment:
I haven't figured out how to do "ordered" here. Right now it's ordered in the sense that the list of providers add suggestions in order of providers in `SUGGEST_PROVIDERS`. I think I might keep it that way, but it does create some scaling problems.

What happens if we get 20 suggestions back? We can't show them all on the Thank You page, so we'll have to trim the list. Do we want the 5 most important ones? If so, how do we order by importance? Etc.

Pushing most of that off for now since we're no where near that scenario, yet.